src/plugin.ts

```diff
@@ -21,7 +21,7 @@ export function buildEnd(err?: Error): void {
 }
 
 function useStyleSheetTransform(filename: string) {
-  return filename.match(/\.s?(a|c)ss/);
```

Comment:
This ends up matching this [third party code](https://github.com/adobe/spectrum-web-components/blob/main/packages/overlay/src/OverlayTrigger.ts#L39)